ci/jobs/satellite6-python-trigger.yaml

```diff
@@ -0,0 +1,88 @@
+                # Change the remote job name here.
+                trigger_type = os.environ.get('TRIGGER_TYPE')
+                remote_job_name = 'elyezer-sandbox'
+                # Note: if trigger_type is 'installation'? False
+                if trigger_type == 'installation':
+                    remote_job_name = 'satellite6-installer'
+                elif trigger_type == 'robottelo':
+                    remote_job_name = 'satellite6-standalone-automation'
+
+                logging.captureWarnings(True)
+                print('Get crumb from Satellite Jenkins server...')
+                crumb = requests.get(
+                    JENKINS_URL + '/crumbIssuer/api/json',
+                    auth=(JENKINS_USERNAME, JENKINS_API_TOKEN),
+                    verify=False
+                ).json()
+                print('Trigger the remote Jenkins job {}...'.format(remote_job_name))
+                queue_url = requests.post(
+                    JENKINS_URL + '/job/{}/buildWithParameters'
+                    .format(remote_job_name),
+                    auth=(JENKINS_USERNAME, JENKINS_API_TOKEN),
+                    data={
+                        'SERVER_HOSTNAME': os.environ.get('SERVER_HOSTNAME'),
+                        'DISTRIBUTION': os.environ.get('DISTRIBUTION', 'DOWNSTREAM'),
+                        'SATELLITE_RELEASE': os.environ.get('SATELLITE_RELEASE', '6.2'),
+                        'SATELLITE_VERSION': os.environ.get('SATELLITE_VERSION', 'GA'),
+                        'SETUP_FAKE_MANIFEST_CERTIFICATE': True,
+                        'TEST_TYPE': os.environ.get('SATELLITE_TEST_TYPE', 'endtoend-api'),
+                        'REMOTE_JENKINS_USERNAME': os.environ.get('REMOTE_JENKINS_USERNAME'),
+                        'REMOTE_JENKINS_API_TOKEN': os.environ.get('REMOTE_JENKINS_API_TOKEN'),
+                        'REMOTE_JENKINS_URL': os.environ.get('REMOTE_JENKINS_URL'),
+                        crumb['crumbRequestField']: crumb['crumb']
+                    },
+                    verify=False
+                ).headers['Location']
+
+                time.sleep(5)
+                queue = requests.get(queue_url + '/api/json', verify=False).json()
+                while not queue.get('executable'):
+                    time.sleep(1)
+                    queue = requests.get(queue_url + '/api/json', verify=False).json()
+                print('Job #{0} ({1}) was triggered...'.format(
+                    queue['executable']['number'],
+                    queue['executable']['url']
+                ))
+
+                result = None
+                while result is None:
+                    time.sleep(1)
+                    response = requests.get(
+                        JENKINS_URL + '/job/{0}/{1}/api/json'
+                        .format(
+                            remote_job_name,
+                            queue.get('executable').get('number')
```

Comment:
The executable and number keys must be defined, replace to `queue['executable']['number']` so it can raise an exception if that is not the case.